plenum/test/node_request/message_request/test_node_requests_missing_preprepare.py

```diff
@@ -53,11 +54,13 @@ def test_node_requests_missing_preprepare(looper, txnPoolNodeSet,
 
     # More than `f` nodes received `MessageReq`
```

Comment:
The comment is wrong. No non-primaries received MessageReq

Reply:
done